modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/events.js

```diff
@@ -35,18 +35,26 @@
 			CLICK_EVENTS[guid] = fn;
 
 			if (!Liferay._baseDelegateHandle) {
-				Liferay._baseDelegateHandle = A.getBody().delegate(
-					'click',
-					Liferay._baseDelegate,
-					'.lfr-delegate-click'
-				);
+				document.body.addEventListener('click', (event) => {
+					const delegateTarget = event.target.closest(
+						'.lfr-delegate-click'
+					);
+
+					if (delegateTarget) {
+						event.delegateTarget = delegateTarget;
+
+						Liferay._baseDelegate(event);
+					}
+				});
+
+				Liferay._baseDelegateHandle = true;
```

Comment:
I think we should use this from dxp, [delegate](https://github.com/liferay/liferay-portal/blob/0849b75262d3a8c2277a5e10a9d822160d602ff7/modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/delegate/delegate.es.js#L45)

Reply:
I wasn't sure how to import it there. When I added it to the top, it threw a lot of errors in the console. So I just went with the regular approach to not spend a lot of time trying to find an example of it working.

Reply:
Yeah I'm not actually too sure how to add that in here appropriately...

I think it might just be best to add an ignore comment for this instance of aui-body and leave the original delegate logic.

Since this logic is fairly core to DXP, I wouldn't want to accidentally cause a regression here.